iclr2016(old scripts)/20160114-demo/web-field-workshop.html

```diff
@@ -1,8 +1,4 @@
 <html>
-  <head>
-    <!--<script src="https://code.jquery.com/jquery-1.10.1.min.js"></script>-->
-    <script src="https://cdn.jsdelivr.net/lodash/3.5.0/lodash.compat.min.js"></script>
```

Comment:
As it works now, once the webfield is loaded via ajax, cajaManager appends the html directly to the dom in either the div#group-container or div#invitation-container depending on the page. Once this happens all scrips are evaluated directly in the context of the current page, with access to all global variables, including $, _, Handlebars.templates, and Open Review globals like controller and view.

This obviously isn't super safe...